server/server.go

```diff
@@ -341,6 +342,12 @@ func New(
 	}
 	hostportManager := hostport.NewHostportManager(iptInterface)
 
+	ip6tInterface := utiliptables.New(utilexec.New(), utiliptables.ProtocolIPv6)
+	if _, err := ip6tInterface.EnsureChain(utiliptables.TableNAT, iptablesproxy.KubeMarkMasqChain); err != nil {
+		logrus.Warnf("unable to ensure ip6tables chain: %v", err)
```

Comment:
Yeah, we should create the host port manager, actually, I don't think the port manager needs this rule, but since it was there before I preferred to keep current implementation.

Also, the port manager checks for the iptables rule here
https://github.com/kubernetes/kubernetes/blob/323f34858de18b862d43c40b2cced65ad8e24052/pkg/kubelet/dockershim/network/hostport/hostport_manager.go#L99

Reply:
I was wrong, the port manager does need the KubeMarkMasqChain. Dan Winship has done an extensive research on this https://github.com/kubernetes/kubernetes/issues/82125

Reply:
anyway, hostportManager must be created despite the iptables rule does not exist